src/webapp/pages/analysis/steps/3-disaggregates/useDisaggregatesStep.tsx

```diff
@@ -0,0 +1,103 @@
+import { ChangeEvent, useState } from "react";
+import i18n from "$/utils/i18n";
+
+export function useDisaggregatesStep() {
+    const catCombosList = [
+        {
+            value: "ACTIVITY",
+            text: i18n.t("Activity Level"),
+        },
+        {
+            value: "SEX",
+            text: i18n.t("Sex"),
+        },
+        {
+            value: "AGE",
+            text: i18n.t("Age Group"),
+        },
+        {
+            value: "AGESEX",
+            text: i18n.t("Age Group + Sex"),
+        },
+        {
+            value: "BIRTHPLACE",
+            text: i18n.t("Place of Birth"),
+        },
+        {
+            value: "TRAININGPLACE",
+            text: i18n.t("Place of Training"),
+        },
+        {
+            value: "FOREIGNTRAINED",
+            text: i18n.t("Foreign Trained"),
+        },
+        {
+            value: "OWNERSHIP",
+            text: i18n.t("Ownership"),
+        },
+        {
+            value: "WORKINGFACILITYTYPE",
+            text: i18n.t("Working Facility Type"),
+        },
+        {
+            value: "NADOMESTICTHWF",
+            text: i18n.t("Newly Active domestic trained HWF"),
+        },
+        {
+            value: "NAFOREIGNTE",
+            text: i18n.t("Newly Active foreign trained employed"),
+        },
+        {
+            value: "VOLUNTARY",
+            text: i18n.t("Voluntary Exits"),
+        },
+        {
+            value: "INVOLUNTARY",
+            text: i18n.t("Involuntary exits"),
+        },
+        {
+            value: "VACANCY",
+            text: i18n.t("Vacancy rate"),
+        },
+        {
+            value: "CONTRACT",
+            text: i18n.t("Contract Type"),
+        },
+        {
+            value: "APPLICATIONS",
+            text: i18n.t("Applications"),
+        },
+        {
+            value: "ENROLLED",
+            text: i18n.t("Enrolled"),
+        },
+        {
+            value: "GRADUATESGENDER",
+            text: i18n.t("Graduates by Gender"),
+        },
+        {
+            value: "GRADUATESINSTITUTION",
+            text: i18n.t("Graduates by institution ownership"),
+        },
+    ];
+
+    const defaultValues = catCombosList.map(option => option.text);
+
+    const [value, setValue] = useState<string[]>(defaultValues);
+
+    const handleChange = (event: ChangeEvent<any>) => {
+        const selectedValues = event.target.value as string[];
+        setValue(selectedValues);
```

Comment:
Now we can work directly with the selected values:

````tsx
const handleChange = (values: string[]) => {
  setValue(values);
};
```

this change apply for all the other hooks as well